server.stats.disk-usage

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -eu
+
+df --output=used / | tail -n1
```

Comment:
The "used" space does not really give a reference as to how far we are from a disaster... I'd suggest perhaps to display the "pcent" or "avail"?

Reply:
I'm actually setting the [maximum value of the gauge](https://github.com/c4dt/stats/blob/add-stats-disk-usage/dashboards-to-provision/services.json#L1600) to show how much "in the red" we are.

Reply:
I'm afraid this will make it a bit brittle in case we change the size of the harddisk later. Can you at least add a comment here, and see if you can put an 'info' or something on the stats?

Reply:
Alright, I've updated to use `--output=pcent`.